### ISSUE TITLE
[HxTabPanel] Add a built-in scroll functionality 

### ISSUE THREAD

**Issue by leonardopugliese** [havit/Havit.Blazor#553]
It would be very convenient to add a built-in scroll functionality. Scroll buttons should automatically appear when there is not enough space available for all tabs

**Comment by crdo** [comment]
Hi, what do you mean by "scroll buttons"?
If you want the content to scroll when overflowing just add `overflow-auto` utility class via `ContentCssClass`.

**Comment by crdo** [comment]
It is going to be ugly on with the Windows scrollbars though...
<img width="686" alt="image" src="https://github.com/havit/Havit.Blazor/assets/1410922/fa7eba9b-e554-4150-80ab-4d125725591c">

**Comment by hakenr** [comment]
I think the request is for something like
![image](https://github.com/havit/Havit.Blazor/assets/12828888/566c10ca-4e56-41be-a472-ed07c8b7810b)

(borrowed from https://ej2.syncfusion.com/react/documentation/tab/adaptive)

**Comment by leonardopugliese** [comment]
> Penso che la richiesta sia per qualcosa di simile ![Immagine](https://user-images.githubusercontent.com/12828888/245067869-566c10ca-4e56-41be-a472-ed07c8b7810b.png)
> 
> (preso in prestito da https://ej2.syncfusion.com/react/documentation/tab/adaptive )

Exactly. Scroll buttons should automatically appear when there is not enough space available for all tabs

**Comment by crdo** [comment]
Got it. 
Given the fact that you need a JS to know if the content overflows or not, I think this issue should be submitted to the Bootstrap repository.
